index.html

```diff
@@ -71,5 +71,87 @@ <h2>Software Developer</h2>
 
        </div>
    </section>
+
+   <section class="middle-section-work">
+       <div class="middle-section">
+           <h3>My recent work</h3>
+           <hr class="first-line">
+           <div class="stories">
+               <div class="story-left">
+                    <div class="rect"></div>
+                    <div class="desc">
+                        <h4>Multi-Purpose Stories</h4>
+                        <p>A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. has been the industry's 
+                            standard dummy text ever since the 1500s, when an unknown 
+                            printer took a standard dummy text.
+                        </p>
+                        <div class="skills">
+                            <p>css</p>
+                            <p>html</p>
+                            <p>bootstrap</p>
+                            <p>Ruby</p>
+                        </div>
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+                <div class="story-left invert">
+                     <div class="desc">
+                         <h4>Multi-Purpose Stories</h4>
+                         <p>A daily selection of privately personalized reads; 
+                             no accounts or sign-ups required. has been the industry's 
+                             standard dummy text ever since the 1500s, when an unknown 
+                             printer took a standard dummy text.
+                         </p>
+                         <div class="skills">
+                             <p>css</p>
+                             <p>html</p>
+                             <p>bootstrap</p>
+                             <p>Ruby</p>
```

Comment:
[Required] Each project technology needs to be nested in the `<ul>` tag and each technology with `<li>` tag. 👍🏼